docs/quick_tutorial/static_assets.rst

```diff
@@ -43,13 +43,21 @@ Steps
    .. literalinclude:: static_assets/tutorial/static/app.css
     :language: css
 
-#. Make sure we haven't broken any existing code by running the tests:
+#. We add a functional test that asserts that the newly added static file is delivered:
+
+   .. literalinclude:: static_assets/tutorial/tests.py
+    :lines: 46-48
+    :lineno-match:
+    :linenos:
+    :language: python
```

Comment:
Sorry for the confusion. This should be:

```
.. literalinclude:: static_assets/tutorial/tests.py
    :language: python
    :pyobject: TutorialFunctionalTests.test_css
    :lineno-match:
```

`:lineno-match:` should replace `:lines:` and `:linenos:`.